components/statistics/ProfileProgress.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```

Comment:
I think this import is not used?

Reply:
Yes I removed it. It came from the snippet I use to create the component.